package.json

```diff
@@ -48,7 +47,7 @@
     "@ethersproject/experimental": "^5.4.0",
     "@ethersproject/hardware-wallets": "github:mudgen/ethers-ledger",
     "@nomiclabs/hardhat-etherscan": "^3.0.0",
-    "@openzeppelin/contracts": "^4.5.0",
+    "@openzeppelin/contracts": "^4.7.0",
```

Comment:
there was a specific contract I was initially inheriting from that was not in 4.5. However I have since removed that inheritance so can downgrade this back to 4.5